agents/langgraph/templates/agentic_rag/Makefile

```
@@ -182,6 +182,199 @@ dry-run: _check-env ## Render Helm templates without deploying
+	  MTLS_DIR=~/.config/openshell/gateways/openshift/mtls && mkdir -p $$MTLS_DIR && \
+	  oc -n openshell get secret openshell-client-tls -o jsonpath='{.data.ca\.crt}'  | base64 -d > $$MTLS_DIR/ca.crt && \
+	  oc -n openshell get secret openshell-client-tls -o jsonpath='{.data.tls\.crt}' | base64 -d > $$MTLS_DIR/tls.crt && \
+	  oc -n openshell get secret openshell-client-tls -o jsonpath='{.data.tls\.key}' | base64 -d > $$MTLS_DIR/tls.key && \
+	  openshell gateway add "$$GATEWAY_URL" --local --name openshift && \
+	  echo "==> Verifying connection..." && \
+	  export OPENSHELL_GATEWAY_INSECURE=true && \
+	  openshell status && \
+	  echo "" && \
+	  echo "Gateway ready. Next: make build-openshell"
+
+teardown-gateway: ## Uninstall openShell gateway and clean up
```

checkmake (0.3.2)

[warning] Target body for "teardown-gateway" exceeds allowed length of 5 lines (6).

(maxbodylength)